src/Query.ts

```diff
@@ -101,15 +100,17 @@ export class Query<T = unknown> implements PromiseLike<T> {
       this.store.__cacheResponse(this.cacheKey, data)
     }
 
-    const value = getFirstValue(data)
     if (this.options.raw) {
       this.loading = false
       this.data = data
       this.onResolve(this.data!)
     } else {
       try {
         this.loading = false
-        const normalized = this.store.merge(value)
+        const normalized: any = {}
```

Comment:
So really not sure how much the typing here matters if at all, but i guess we could at least make it:

```ts
const normalized: { [key: string]: any } = {}
```